app/controllers/webhook_controller.rb

```diff
@@ -35,6 +36,42 @@ def callback
             message = "登録ありがとうございました！チケットはこちらになります\n" + tickets['url']
             LineApi.push_message(line_user_id, message)
           end
+
+          message = "会員登録をすると更にチケットが貰えます！\n" + "会員登録をするためには、「会員登録　生年月日 性別」(例：会員登録　1999/1/27 男)と入力してください" 
+          LineApi.push_message(line_user_id, message)
+        end
+      when Line::Bot::Event::Message
+        case event.type
+        when Line::Bot::Event::MessageType::Text
+          input_text = event.message['text']
+          if p input_text.start_with?("会員登録")
```

Comment:
[nits]
デバッグ用の`p`は消しておいた方がいいですね〜
```suggestion
          if input_text.start_with?("会員登録")
```